compiler/README.md

```diff
@@ -0,0 +1,73 @@
+<!-- README Document on how to build and use this Neural Network Compiler -->
+
+# Introduction
+
+This document discusses the ONE compiler. ONE compiler analyses, quantizes and optimizes the input model. It takes many model formats and converts to a common [Circle](#circle-format)  format which is then deployed.
+
+![Diagram](./drawing.svg)
+
+<!-- Contents -->
+# Contents 
+* [Build Setup](#Build-Setup)
+* [Supported Formats](#Supported-Nerual-Network-Formats)
+* [Model Analysis](#Model-Analysis)
+* [Data Representation & Quantization](#Data-Representation-And-Quantization)
+* [Circle Format](#Circle-Format)
+
+<!-- Build Setup -->
+## Build Setup
+<!-- System Requirments -->
+* System Setup   
+This setup is to build the ONE compiler on Ubuntu 18.04.   
+Get the code [Samsung ONE](https://github.com/Samsung/ONE) from Github. 
+
+<!-- Software/Package Requirements -->
+* Dependencies
+    * Cmake (3.10.2)
+    * g++ toolchain (9.0.1)
+    * python virtual environment (python3-venv for python3)
+    * HD5 library (libhdf5-dev)
```

Comment:
About the versions, our current CI is Ubuntu 16.04 and from my linux which is same 16.04 has;
- cmake 3.5.1
- g++ 5.5.0

Reply:
> The image, https://github.com/Samsung/ONE/raw/ba675262d626316165439abef251ef8757a101de/compiler/drawing.svg,
> the word `conversion kit` is new to me and I cannot see any explanation.
> And the diagram doesn't seem to be correct. ONE compiler input is TF, ONNX, TFlite, and maybe Caffe (not sure Caffe is converted to circle).
> Output is circle.

@seanshpark I see Caffe to TFLite converter in directory. So effectively Caffe -> TFLite -> [ONE Compiler] gives output as Circle. If this is NOT possible, let me know.

Reply:
@seanshpark These are the ones I tested with.